pkg/policy/api/zz_generated.deepcopy.go

```diff
@@ -11,7 +11,7 @@ package api
 import (
 	labels "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/labels"
 	v1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
-	kafka "github.com/cilium/cilium/pkg/policy/api/kafka"
+	kafka "github.com/cilium/proxy/pkg/policy/api/kafka"
```

Comment:
Was this line manually changed? Or did you use the Makefile to change this?

Reply:
it was changed manually, find and replace in IDE helps a lot though.